packages/block-template-html/.npmignore

```diff
@@ -0,0 +1,2 @@
+/dev/**/*
```

Comment:
`/dev/index.html` needs to be included (and restored), as it is required by the `dev` command

Reply:
Updated in https://github.com/blockprotocol/blockprotocol/pull/480/commits/834f5c0a93bd2081a38781e6e4977649948409a5